.gitignore

```diff
@@ -1,3 +1,4 @@
+*.zip
```

Comment:
We actually have a few examples where we need those zips, as in: https://github.com/terraform-providers/terraform-provider-aws/tree/master/examples/cognito-user-pool so we should remov this one

Reply:
I will fix that mistake.